test/conftest.py

```diff
@@ -44,6 +45,7 @@ def objdict() -> T.Dict[str, T.Any]:
         "edges": {},
         "subgraphs": {},
     }
+    return copy.deepcopy(obj)
```

Comment:
- And this fixture now makes a copy of the dictionary it supplies each time, because if it's passing the same dictionary object reference to each test, they'll stomp all over each other modifying it.